0-prereqs.md

```diff
@@ -2,11 +2,12 @@
 
 You will need to install the following to follow along with this tutorial:
 
-- Go v1.18
+- [Go v1.18](https://go.dev/dl/)
```

Comment:
I am using Go v1.19 as cluster-api uses 1.19. Is our recommendation still Go v1.18, @richardcase?

Reply:
We can change it to 1.19

Reply:
done